source/wp-content/themes/wporg-developer-2023/src/code-section-title/block.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
What's the purpose of this block that the heading block doesn't handle? I'm not opposed to adding things & then iterating on them, but I'd rather start lighter - removing stuff later often doesn't happen.

Reply:
Since the titles will require an anchor link, I thought it made sense to stub it out.

<img width="184" alt="Screen Shot 2023-01-13 at 8 41 24 AM" src="https://user-images.githubusercontent.com/1657336/212204353-6874a58e-0134-4982-80a7-49ba727af891.png">

Reply:
That's built in to the Table of Contents block, it works on any heading block (I can move that block into mu-plugins so we can share it here).

Reply:
Great. I'll remove the custom block.

Reply:
I think about it differently - using headings gives us more freedom to change the heading architecture in the future, if we wanted to break up the sections into subsections or something (like, post reskin-redesign).

> We could also remove the headings from the blocks and conceive a different way to conditionally render sections.

That's an interesting idea… since that's the reason you're adding them in the block instead of putting the heading in the pattern? That would be a good future iteration, maybe title + level as an attribute and a helper function all the blocks can use to render it? or some use of InnerBlocks with only a heading block… anyway, that's getting ahead of things 😁